### DISCUSSION TITLE
LJSpeech training result

### DISCUSSION THREAD

**Discussion by WendongGan** [p0p4k/vits2_pytorch#29] [Show and tell]
@p0p4k hi , p0p4k, thanks again for your shared code. 
![QVkWRiLbq5](https://github.com/p0p4k/vits2_pytorch/assets/40649244/4f64a9ce-f165-48f9-8129-c5e0ef5004ee)

I have trained the two configure in LJSpeech  about 300k+ steps , with batch size 32, 3090Ti
# LJ Speech
python train.py -c configs/vits2_ljs_nosdp.json -m ljs_base # no-sdp; 
python train.py -c configs/vits2_ljs_base.json -m ljs_base # with sdp;

Result samples
[VITS2_LJSpeech Result_from WendongGan-2023.08.29.zip](https://github.com/p0p4k/vits2_pytorch/files/12459992/VITS2_LJSpeech.Result_from.WendongGan-2023.08.29.zip)

Pretrained model:
(coming soon.....)

Now, We think no-dsp is better, maybe it is more easy to train. When i finish my training, i will update the samples

**Comment by p0p4k** [comment]
Thanks for the samples and the update. Do you think it is better than VITS-1?

**Reply by WendongGan** [reply]
yes, I think it's better than VTS-1 in terms of emphasis and pauses. What do you think? @p0p4k

**Reply by p0p4k** [reply]
> yes, I think it's better than VTS-1 in terms of emphasis and pauses. What do you think? @p0p4k 

I think so too!

**Reply by harmlessman** [reply]
How is the synthesis speed compared to vits1? Is the vits2 much faster?

**Comment by EobardThawne721** [comment]
> @p0p4k hi , p0p4k, thanks again for your shared code. ![QVkWRiLbq5](https://private-user-images.githubusercontent.com/40649244/263960088-4f64a9ce-f165-48f9-8129-c5e0ef5004ee.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3OTAyOTAsIm5iZiI6MTcxMzc4OTk5MCwicGF0aCI6Ii80MDY0OTI0NC8yNjM5NjAwODgtNGY2NGE5Y2UtZjE2NS00OGY5LTgxMjktYzVlMGVmNTAwNGVlLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDEyNDYzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkMGQyN2NiZWNiMGI5YzhjYjBjNTRlNDBmZmIxY2Q0YzM5Njk0Y2YyYTljMDUyY2NlMjJmNTVlMDMxYjk3ZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zLceuBZ4O4my5OJSFkeoeN7ksdW2o5tIrb2UCw1ptp8)
> 
> I have trained the two configure in LJSpeech about 300k+ steps , with batch size 32, 3090Ti
> 
> # LJ Speech
> python train.py -c configs/vits2_ljs_nosdp.json -m ljs_base # no-sdp; python train.py -c configs/vits2_ljs_base.json -m ljs_base # with sdp;
> 
> Result samples [VITS2_LJSpeech Result_from WendongGan-2023.08.29.zip](https://github.com/p0p4k/vits2_pytorch/files/12459992/VITS2_LJSpeech.Result_from.WendongGan-2023.08.29.zip)
> 
> Pretrained model: (coming soon.....)
> 
> Now, We think no-dsp is better, maybe it is more easy to train. When i finish my training, i will update the samples

Hello, why is it that I've run 450k steps on a single GPU using the code provided, only changing the batch size to 8, but still see no improvement?（你好，为什么我用他提供的代码，我只更改了batch为8，在单个gpu上运行了450k步还是没效果呢）

**Comment by JohnHerry** [comment]
You should better adjust your batch size.  the value of 8 is too small. and, multi-card training helps the smooth traning。

**Reply by EobardThawne721** [reply]
Can it only run on multiple GPUs? Can't it run on a single GPU? What is the usual batch size for multi-GPU training？
